ui/app/AppLayouts/Wallet/controls/SortOrderComboBox.qml

```diff
@@ -220,6 +220,9 @@ ComboBox {
         highlighted: root.highlightedIndex === index
         enabled: !isSeparator
         visible: {
+            if (modelData["isDisabled"]) {
```

Comment:
Maybe it's not the scope of this PR, but it would be nice to use regular model for `SortOrderComboBox`. It would give ability to filter externally, and that component could be unaware of any specific criteria for visibility. Also hiding delegates is in general not recommended approach - the filtering should be done on a model level.

Reply:
you are correct.
In the meantime, I believe it's important for the combo box to provide a way to enable/disable item interaction.  Currently, setting enable: false prevents selection but doesn't offer clear visual feedback. To prioritize user experience, I've opted to hide disabled items for now.

I'll also reached out to our design team to establish a clear concept for visually disabled items. Once we have their guidance, we can confidently implement the disable property to provide a more polished and intuitive solution